ui/components/app/wallet-overview/aggregated-percentage-overview.test.tsx

```diff
@@ -484,14 +484,14 @@ describe('AggregatedPercentageOverview', () => {
       },
     });
     const expectedAmountChange = '-$0.39';
-    const expectedPercentageChange = '(-1.09%)';
+    const expectedPercentageChange = '(-1.08%)';
```

Comment:
Just making sure that this change is expected

Reply:
I think this test was skipped in first push because i mistakenly left an "only" thats why pipeline was green and i missed it;